packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -307,8 +307,15 @@ resource.addMethod('GET', integration, {
   requestParameters: {
     'method.request.querystring.who': true
   },
-  // We need to set the validator for ensuring they are passed
+  // To specify a request validator we can set either `requestValidator` or `requestValidatorOptions`
+  // We can set the validator for ensuring they are passed
   requestValidator: validator,
+  // Or we can set request validator options like below
+  requestValidatorOptions: {
+    requestValidatorName: 'test-validator',
+    validateRequestBody: true,
+    validateRequestParameters: false
+  }
```

Comment:
This gets confusing. Instead, keep the `requestValidatorOptions` here and drop `requestValidator`.

Add a paragraph after this code block stating something like - "Specifying `requestValidatorOptions` automatically creates the `RequestValidator` construct with the given options. However, if you have your `RequestValidator` already initialized or imported, use the `requestValidator` option instead."

Reply:
Done